searx/plugins/limiter.py

```diff
@@ -62,6 +67,10 @@ def is_accepted_request() -> bool:
             logger.debug("BLOCK %s: got Connection=close", x_forwarded_for)
             return False
 
+        if not request.headers.get('Connection'):
+            logger.debug("BLOCK %s: got Connection=", x_forwarded_for)
```

Comment:
AFAIK the connection header is not used in http2 -> https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Connection . IMO we should not filter for this header.

Reply:
I'm also against this change, I don't think it's wise to block based on this missing HTTP header.

Reply:
Thanks for having an eye on .. I removed `if not request.headers.get('Connection'):`.